### ISSUE TITLE
New card state: not enough storage state

### ISSUE THREAD

**Issue by filopedraz** [premAI-io/prem-app#219]
### Description

It can happen that the user doesn't have enough storage in order to download the service. In that case, we need to disable the download as we do for memory. The message should be `You don't have enough hard disk available, clean your storage, if you wanna download the current service.`

In order to handle this condition, you can use the `dockerImageSize` property available for each service.

**Comment by jigneshsolanki** [comment]
@filopedraz dependent on [#96](https://github.com/premAI-io/prem-daemon/issues/96)
